### PR TITLE
add Vue.js `sariasan.com` free full lessons

### PR DESCRIPTION
**Pull request by Amirrezaheydari81**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Amirrezaheydari81** [comment]
add vue.js website free lean for line 80

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.
I think this should go on the courses page, not the books page.
If you add a category, also add it to the index at the top of the page.

File: books/free-programming-books-fa_IR.md [hunk 1]
```diff
@@ -77,4 +77,7 @@
 * [راهنمای زبان R](http://cran.r-project.org/doc/contrib/Mousavi-R-lang_in_Farsi.pdf) (PDF)
 * [موضعات ویژه در R](http://cran.r-project.org/doc/contrib/Mousavi-R_topics_in_Farsi.pdf) (PDF)
 
+### Vue.js
+* [آموزش Vue.js از صفر تا صد با 8 درس رایگان](https://sariasan.com/featured/vue-free-full-lessons)
+
```
Comment (davorpa) [inline books/free-programming-books-fa_IR.md:83]: Only one blank line at the end of document

```suggestion

```

File: books/free-programming-books-fa_IR.md [hunk 2]
```diff
@@ -77,4 +77,5 @@
 * [راهنمای زبان R](http://cran.r-project.org/doc/contrib/Mousavi-R-lang_in_Farsi.pdf) (PDF)
 * [موضعات ویژه در R](http://cran.r-project.org/doc/contrib/Mousavi-R_topics_in_Farsi.pdf) (PDF)
 
-
+### Vue.js
+* [آموزش Vue.js از صفر تا صد با 8 درس رایگان](https://sariasan.com/featured/vue-free-full-lessons)
```
Comment (davorpa) [inline books/free-programming-books-fa_IR.md:81]: Solve linter faults:

```
/free-programming-books/books/free-programming-books-fa_IR.md
76:1-78:96  warning  Incorrect number of blank lines between last section and next heading  blank-lines-1-0-2  remark-lint
80:1-80:11  warning  Incorrect number of blank lines between heading and section            blank-lines-1-0-2  remark-lint

⚠ 2 warnings
```

Apply following patch :arrow_down:
```suggestion

### Vue.js

* [آموزش Vue.js از صفر تا صد با 8 درس رایگان](https://sariasan.com/featured/vue-free-full-lessons)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Like Eric says in https://github.com/EbookFoundation/free-programming-books/pull/6917#issuecomment-1184350008...

- [x] Linter faults was solved :rocket:
- [ ] See if this resource fits better in course listing
- [x] Adds the corresponding entry in the TOC (see at the documment beginning) also preserving the alphabetization of said section / list. **Added at 09f5ebb4a40f756341fba6105a4f7de06e9df3c0**

**Comment by davorpa** [comment]
Conflicts are with #6988 which introduces some indentation at TOC section

**Comment by davorpa** [comment]
> * See if this resource fits better in course listing

**NOTE** This definitely goes in courses since the resource has videos that cannot be kept as part of it printed version

**Review by davorpa — CHANGES_REQUESTED** [review]
> * See if this resource fits better in course listing

**NOTE** This definitely goes in `courses/free-courses-fa_IR.md` since the resource has videos that cannot be kept as part of it printed version

File: books/free-programming-books-fa_IR.md [hunk 3]
```diff
@@ -83,3 +84,8 @@
 * [تحلیل شبکه‌های اجتماعی در R](http://cran.r-project.org/doc/contrib/Raeesi-SNA_in_R_in_Farsi.pdf) (PDF)
 * [راهنمای زبان R](http://cran.r-project.org/doc/contrib/Mousavi-R-lang_in_Farsi.pdf) (PDF)
 * [موضعات ویژه در R](http://cran.r-project.org/doc/contrib/Mousavi-R_topics_in_Farsi.pdf) (PDF)
+
+
+### Vue.js
+
+* [آموزش Vue.js از صفر تا صد با 8 درس رایگان](https://sariasan.com/featured/vue-free-full-lessons)
```
Comment (davorpa) [inline books/free-programming-books-fa_IR.md:91]: Add resource author

```suggestion
* [آموزش Vue.js از صفر تا صد با 8 درس رایگان](https://sariasan.com/featured/vue-free-full-lessons) - میلاد حیدری
```

**Review by davorpa — APPROVED** [review]

**Comment by davorpa** [comment]
@eshellman I move it to courses in 4f1086d09cd596fa84f0e24e9432e7fb8372f1fc so all is resolved

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**